### DISCUSSION TITLE
Displaying Header Links of All Pages

### DISCUSSION THREAD

**Discussion by joshuacurtiss** [vuepress/core#1029] [Q&A] [answered]
Hello everyone, is there any way to accomplish the v1 equivalent of [Displaying Header Links of All Pages](https://vuepress.vuejs.org/theme/default-theme-config.html#displaying-header-links-of-all-pages) for the sidebar?  I can't find anything acknowledging its absence, so not sure what your stance on it is.  As I upgrade some projects to VuePress 2, that is a functionality I have not been able to duplicate. Thanks for your thoughts.

Josh

**Comment by Mister-Hope** [comment, marked as answer]
To improve scalability, all pages data is not avaiable now, which means you can only visit current page data by default, so we are no longer providing display all headers.

**Reply by joshuacurtiss** [reply]
Thanks.
